examples/plotting/masked_composite_plot.py

```diff
@@ -0,0 +1,61 @@
+"""
+================================
+Combining off-limb and disc maps
+================================
+
+This example combines creating a composite plot with masking out the
+solar disk. The purpose is to introduce users to the plotting process
+required to overlay multiple Maps that have been modified with other
```

Comment:
```suggestion
required to overlay multiple maps that have been modified with other
```
I can never remember what's correct, is it this or Maps?

Reply:
Looking at other examples and documentation, it seems like maps is more commonly used than Maps.